content/pages/use-cases/ai-agents.md

```diff
@@ -1,120 +1,45 @@
-## AI agents are now provisioning more databases on Neon than humans—many thousands per day.
-
----
+## AI Agents are now provisioning 80% of Neon databases (tens of thousands per day)
 
 The scale is massive, and Neon is built to handle it.
 
-### Purpose-built interfaces for AI Agents.
-
-Neon offers dedicated interfaces that make it easy for AI agents to deploy and manage databases:
-
-**[Model Context Protocol (MCP) server](https://github.com/neondatabase-labs/mcp-server-neon):** Enables any MCP Client to interact with Neon's API using natural language. AI agents can use Neon's MCP server to automate tasks such as creating databases, running SQL queries, and managing database migrations. [Explore our MCP guides](/blog?query=MCP).
-
-**[@neondatabase/toolkit](https://github.com/neondatabase/toolkit):** A lightweight client designed for AI agents that need to spin up Postgres databases in seconds and run SQL queries. It includes both the Neon TypeScript SDK and the Neon Serverless Driver.
-
-```jsx showLineNumbers
-import { NeonToolkit } from "@neondatabase/toolkit";
-
-const toolkit = new NeonToolkit(process.env.NEON_API_KEY!);
-const project = await toolkit.createProject();
-
-await toolkit.sql(
-  project,
-  `
-    CREATE TABLE IF NOT EXISTS
-      users (
-          id UUID PRIMARY KEY,
-          name VARCHAR(255) NOT NULL
-      );
-  `,
-);
-await toolkit.sql(
-  project,
-  `INSERT INTO users (id, name) VALUES (gen_random_uuid(), 'Sam Smith')`,
-);
-
-console.log(await toolkit.sql(project, `SELECT name FROM users`));
-
-await toolkit.deleteProject(project);
-
-```
-
-### Control resource consumption at scale.
-
-Managing thousands of databases requires predictable resource allocation. With Neon’s API, you can:
-
-- Set limits on compute uptime, CPU usage, data writes, storage, and data transfer
-- Define different quota tiers (e.g. for free, pro, and enterprise plans)
-
-### Define compute configuration.
+The [Neon MCP server](https://mcp.neon.tech/) enables any client to interact with Neon's API using natural language. AI agents can use Neon's MCP server to automate tasks such as creating databases, running SQL queries, and managing database migrations. 
 
-AI agents need flexibility in how they allocate and scale database resources. Neon enables precise compute management:
+The [@neondatabase/toolkit](https://github.com/neondatabase/toolkit) is a lightweight client designed for AI agents that need to spin up Postgres databases in seconds and run SQL queries. It includes both the Neon TypeScript SDK and the Neon Serverless Driver.
 
-- Configure autoscaling limits to control min/max CPU allocation
-- Adjust scale to zero behavior
+<QuoteBlock name="lincoln" className="mt-10" />
 
-### Monitor the fleet.
+<CTA title="Meet app.build, our reference agent" description="app.build is an open-source AI agent, serving as a reference implementation for any agent builder using Neon." buttonText="Explore the code" buttonUrl="/contact-sales" />
```

Comment:
It appears "contact-sales" isn't intentional, perhaps https://github.com/appdotbuild/?